pages/url-encode.tsx

```diff
@@ -126,16 +131,32 @@ export default function URLEncodeDecode() {
             label='Encoded'
             value={encoded}
             onChange={(e) => {
-              setDecoded(decodeURIComponent(e.target.value));
               setEncoded(e.target.value);
+
+              try {
+                setDecoded(decodeURIComponent(e.target.value));
+                setDecodeError(false);
+              } catch {
+                setDecodeError(true);
+              }
             }}
             multiline
           />
           <Box
             display='flex'
+            alignItems='center'
             flexWrap='wrap'
             justifyContent='end'
           >
+            {decodeError && (
```

Comment:
Can we put this on its own below the buttons? Because at 375px width for example, the error message makes the buttons jump around.

Reply:
Sure thing!

Reply:
![image](https://user-images.githubusercontent.com/26071571/194384400-8d3d9eaa-c610-4a06-9aba-0e4102fb2065.png)

Reply:
Excellent, thanks!